### DISCUSSION TITLE
quiero ser sponsor

### DISCUSSION THREAD

**Discussion by stxgho** [community/community#69072] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hola comunidad, quiero ser sponsor pero lleva 4 semanas que envié la solicitud. es normal?

**Comment by 3LIAS-MB** [comment]
Buenas noches. Aproximadamente se tarda 4 meses en constarte.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
